server/app/Http/Controllers/UserController.php

```diff
@@ -636,7 +633,7 @@ public function oauthLogin($user_info, $redirect = false)
         }
     }
 
-    public function getSupport($user, $doc)
+    public function getSupport(DocAccessReadRequest $request, $user, Doc $doc)
     {
         $docMeta = DocMeta::where('user_id', $user->id)->where('meta_key', '=', 'support')->where('doc_id', '=', $doc)->first();
```

Comment:
Also a few lines down in the next couple `where` clauses for gathering counts.

It seems weird that we're returning overall support/oppose counts from the UserController in the first place, but, ya know..